src/pages/yujin/Main/Main.js

```diff
@@ -1,7 +1,313 @@
+                    className="post_profile_pic"
+                    src="images/yujin/teddy.jpg"
+                  />
+                </div>
+                <a href="#" className="post_profile_name">
+                  <b className="makeIdBold">yuzinnee</b>
+                </a>
+              </div>
+              <div className="post_profile_dot_icon">
+                <a href="#">
+                  <i className="fa-solid fa-ellipsis" />
+                </a>
+              </div>
+            </div>
+            <img
+              className="post_picture"
+              src="images/yujin//newyork.jpg"
+              alt="pic"
+            />
+            <div className="feed_icon">
+              <div className="feed_icon_left">
+                <img src="images/yujin/heart.png" className="feed_icon_heart" />
+                <img src="images/yujin/chat.png" className="feed_icon_chat" />
+                <img
+                  src="images/yujin/direct.png"
+                  className="feed_icon_direct"
+                />
+              </div>
+              <div className="feed_icon_right">
+                <img
+                  src="images/yujin//bookmark.png"
+                  className="feed_icon_bookmark"
+                />
+              </div>
+            </div>
+            <div className="who_like">
+              <div className="who_info">
+                <img className="who_pic" src="images/yujin//teddy.jpg" />
+                <a href="" className="who_name">
+                  <b className="makeIdBold">yuzinnee</b>
+                </a>{' '}
+                님 외<b className="makeIdBold">7,096,816명</b>이 좋아합니다
+              </div>
+            </div>
+            <div className="feed_content">
+              <a href="" className="writer_name">
+                <b className="makeIdBold">yuzinnee</b>
+              </a>{' '}
+              가보고 싶은 뉴욕 🤩 여행 가게 된다면...
+              <button className="more_see">더 보기</button>
+              <div className="when_upload">1일 전</div>
+            </div>
+
+            <div className="comment_box">
+              <div className="comment_box_left">
+                <p className="user_name">mo_eden</p>
+                <div className="feed_comment_content">
+                  뉴욕하면 베이글이지 !!🥯
+                </div>
+              </div>
+              <div className="comment_box_right">
+                <img
+                  className="comment_icon_like"
+                  src="images/yujin/heart.png"
+                />
+              </div>
+            </div>
+            <div className="comment_result" />
+            <div className="feed_comments_write">
+              <input
+                className="feed_comments_write_box"
+                placeholder="    댓글 달기..."
+                type="text"
+              />
+              <button className="comment_upload">
+                <b className="makeIdBold">게시</b>
+              </button>
+            </div>
```

Comment:
```suggestion
            <form className="feed_comments_write">
              <input
                className="feed_comments_write_box"
                placeholder="    댓글 달기..."
                type="text"
              />
              <button className="comment_upload">
                <b className="makeIdBold">게시</b>
              </button>
            </>
```
form태그가 더 시멘틱하고 좋을 것 같아요!